deployments/ocis-nats/README.md

```diff
@@ -0,0 +1,48 @@
+# oCIS with NATS deployment example
+
+## Introduction
+
+This example shows how to deploy NATS with NATS as message bus and store.
+It will deploy an oCIS instance and NATS, preconfigured to work together.
+
+***Note***: This example is not intended for production use. It is intended to get a working oCIS
+with NATS running in Kubernetes as quickly as possible. It is not hardened in any way.
+
+## Getting started
+
+### Prerequisites
+
+This example requires the following things to be installed:
+
+- [Kubernetes](https://kubernetes.io/) cluster, with an ingress controller installed.
+- [Helm](https://helm.sh/) v3
+- [Helmfile](https://github.com/helmfile/helmfile)
+
+### End result
+
+After following the steps in this guide, you should be able to access the following endpoint, you
+may want to add these to your `/etc/hosts` file pointing to your ingress controller IP:
+
+- https://ocis.kube.owncloud.test
+
+Note that if you want to use your own hostname and domain, you will have to change the `externalDomain` value.
```

Comment:
```suggestion
### Final Result

After following the steps in this guide, you should be able to access the following endpoint.
You may want to add these to your `/etc/hosts` file pointing to your ingress controller IP:

- https://ocis.kube.owncloud.test

Note that if you want to use your own hostname and domain, you will have to change the `externalDomain` value.
```
...should be able to access the following endpoint...

but where is the endpoint mentioned?

Reply:
The endpoint is the the url down below, so `https://ocis.kube.owncloud.test`.

Reply:
THX, will adapt the suggestion